gamemodes/terrortown/entities/weapons/weapon_ttt_defuser.lua

```diff
@@ -0,0 +1,97 @@
+if SERVER then
+	AddCSLuaFile()
+end
+
+SWEP.HoldType = "slam"
+
+if CLIENT then
+	SWEP.PrintName = "defuser_name"
+	SWEP.Slot = 7
+
+	SWEP.DrawCrosshair = false
+	SWEP.ViewModelFOV = 10
+
+	SWEP.EquipMenuData = {
+		type = "item_weapon",
+		desc = "defuser_desc"
+	}
+
+	SWEP.Icon = "vgui/ttt/icon_defuser"
+end
+
+SWEP.Base = "weapon_tttbase"
+
+SWEP.ViewModel = "models/weapons/v_crowbar.mdl"
+SWEP.WorldModel = "models/weapons/w_defuser.mdl"
+
+SWEP.Primary.ClipSize = -1
+SWEP.Primary.DefaultClip = -1
+SWEP.Primary.Automatic = true
+SWEP.Primary.Delay = 1
+SWEP.Primary.Ammo = "none"
+
+SWEP.Secondary.ClipSize = -1
+SWEP.Secondary.DefaultClip = -1
+SWEP.Secondary.Automatic = true
+SWEP.Secondary.Ammo = "none"
+SWEP.Secondary.Delay = 2
```

Comment:
I think once we remove the secondary attack this can also be removed.
But since i dont know, you should test that